checkout.css

```diff
@@ -0,0 +1,186 @@
+
+:root {
+	font-size: 62.5%;
+	--primary-color: #ff3f6c;
+	--dark-primary-color: #e63961;
+	--complimentary-text-color: white;
+	--primary-text-color: #3e4152;
+	--secondary-text-color: #696b79;
+	--border-color: #e6e6e6;
+}
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	font-family: "Roboto", sans-serif;
+}
+.container {
+	padding: 3rem 0;
+	text-align: center;
+}
+.checkout-heading {
+	display: inline-block;
+	font-size: 2rem;
+	font-weight: 700;
+	color: #282c3f;
+	padding: 3rem 0;
+}
+.checkout-component {
+	width: 65%;
+	display: flex;
+	margin: 0 auto;
+    height: fit-content;
+}
+.card-items {
+	display: flex;
+	flex-direction: column;
+}
+.card-item {
+	display: flex;
+	width: 100%;
+	border: 2px solid var(--border-color);
+	border-radius: 0.35rem;
+	margin: 1rem 0;
+}
+.image-container {
+	width: 157px;
+	height: 148px;
+}
+.image-res {
+	width: 100%;
+	height: auto;
+}
+.card-description {
+	display: flex;
+	flex-direction: column;
+	width: calc(100% - 130px);
+	padding: 1rem;
+	align-items: start;
+}
+.card-item-heading {
+	font-size: 1.6rem;
+	font-weight: 500;
+	color: #282c3f;
+	margin-bottom: 0.3rem;
+}
+.card-item-desc {
+	font-size: 1.2rem;
+	margin-bottom: 0.3rem;
+}
+.card-item-sold {
+	font-size: 1.2rem;
+	color: #94969f;
+	font-weight: 300;
+}
+.card-item-cost {
+	display: flex;
+	padding: 0.75rem 0;
+}
+.card-item-pricecost {
+	font-weight: 700;
+	font-size: 1.6rem;
+	color: #282c3f;
+	margin-right: 1rem;
+}
+.card-item-actualcost {
+	font-size: 1.6rem;
+	color: #282c3f;
+	margin-right: 1rem;
+	text-decoration: line-through;
+}
+.card-item-offer {
+	font-size: 1.6rem;
+	color: var(--primary-color);
+}
+.card-item-quantity {
+	margin: 0.5rem 0;
+	display: flex;
+	align-items: center;
+}
+.item-quantity-btn {
+	padding: 0.5rem 1rem;
+	font-size: 1.6rem;
+	border: 2px solid #e9e9eb;
+	border-radius: 0.35rem;
+	color: #282c3f;
+	background-color: transparent;
+	outline: none;
+	cursor: pointer;
+	margin: 0.5rem 0;
+	margin-right: 1rem;
+}
+.card-item-num {
+	font-size: 1.6rem;
+	font-weight: 500;
+	margin-right: 1rem;
+}
+.card-buttons {
+	padding: 0.5rem 1rem;
+	background-color: var(--primary-color);
+	color: var(--complimentary-text-color);
+	font-size: 1.6rem;
+	border: 2px solid #e6e6e6;
+	border-radius: 0.5rem;
+}
+
+.card-buttons:hover {
+	background-color: var(--dark-primary-color);
+}
+.card-display-cost {
+	display: flex;
+    flex-direction: column;
+    align-items: start;
+	width: 32%;
+	height: auto;
+	border-left: 2px solid var(--border-color);
+	margin: 1rem 2rem;
+	padding: 0.5rem 1rem;
+}
+.card-cost-heading{
+    font-size: 1.6rem;
+    color: #535766;
+    font-weight: 500;
+    margin-bottom: 1rem;
+}
+.card-cost-detail{
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    margin-bottom: 1rem;
+    font-size: 1.4rem;
+    color: #282c3f;
+    font-weight: 400;
+}
+
+.card-cost-discount{
+    color: #03a685 !important;
```

Comment:
Arranging codes or code files to maintain specificity could be done to avoid using !important

Reply:
I removed important word color property. thanks :)